.github/workflows/ci.yml

```diff
@@ -1,8 +1,8 @@
 on:
   push:
-    branches: [ master ]
+    branches: [ master, SegcacheExp ]
```

Comment:
Can we clean this up?

Reply:
No, it will not break my code. 
I will clean this up in the last commit, so that I can use the test to make sure I do not break anything. 

> If I merge the latest ccommon before your test code would it break you subsequent PR?